xbmc/cores/FFmpeg.cpp

```diff
@@ -140,11 +140,14 @@ FFmpegExtraData::FFmpegExtraData(size_t size)
 {
   if (!m_data)
     throw std::bad_alloc();
+  // E.g. AVCodecParameters expects the padding to be zeroed
+  std::memset(m_data + m_size, 0, AV_INPUT_BUFFER_PADDING_SIZE);
```

Comment:
`av_mallocz` instead?

Reply:
Yeah, probably isn't more expensive to have the full buffer initialized :+1: